test/python/qasm/issue_111.qasm

```diff
@@ -0,0 +1,18 @@
+OPENQASM 2.0;
```

Comment:
Can we rename this file to have a more descriptive name? Something like: test_float_precision.qasm